sphinx/deprecation.py

```diff
@@ -10,12 +10,8 @@
 """
 
 
-class RemovedInSphinx16Warning(DeprecationWarning):
-    pass
-
-
 class RemovedInSphinx17Warning(PendingDeprecationWarning):
```

Comment:
RemovedInSphinx17Warning must be derived from `DeprecationWarning` instead of PendingDeprecationWarning.

And sphinx should have `RemoveInSphinx18Warning(PendingDeprecationWarning)` class.